### PR TITLE
Chocolatey package documentation.

### PR DESCRIPTION
**Pull request by kwilliams1987**
Added Chocolatey package details.

_This PR is a draft until Chocolatey approve the package._

## Proposed Changes
  - Automated install via Chocolatey package manager

## Checklist
No code changes are required, so I've left these empty.

- [ ] Tested on all platforms changed
- [ ] `cargo fmt` has been run on this branch
- [ ] `cargo clippy` has been run on this branch
- [ ] Added an entry to `CHANGELOG.md` if knowledge of this change could be valuable to users

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,12 @@ Prebuilt binaries for macOS and Windows can be downloaded from the [GitHub relea
 
 For everyone else, a detailed instruction can be found [here](https://github.com/casperstorm/ajour/blob/master/INSTALL.md).
 
+### Chocolatey
+
+The app can also be installed via the Chocolatey package manager via `choco install ajour`.
+
+This package is automatically updated to the latest version every day at 06:00 UTC and will add a Start Menu shortcut.
+
```
Comment (casperstorm) [inline README.md:61]: Great addition, can you move it to https://github.com/casperstorm/ajour/blob/master/INSTALL.md#third-party-repositories

**Comment by kwilliams1987** [comment]
The originally submitted `ajour` package (0.3.4) has finally been approved and I've now pushed a 0.4.1 version, however `ajour-opengl` (0.3.5) is still pending review.

**Comment by casperstorm** [comment]
Is it manually work to submit new releases, @kwilliams1987?

**Comment by kwilliams1987** [comment]
> Is it manually work to submit new releases, @kwilliams1987?

I have a Github Action which checks every morning at 06:00 UTC for updates, if it finds one it will update the Chocolatey package, commit it to the Github repository and automatically submit it to Chocolatey for approval.

Currently once submitted to Chocolatey there is a delay of up to a few days before it is released to general public but over time the package should become "trusted" and only require two automated checks (with a delay of only an hour or so).

For you there is no additional work, just keep using your existing release format (1.2.3).

**Comment by kwilliams1987** [comment]
Both packages are now available, Ajour is already 0.4.1, Ajour (OpenGL) is still 0.3.5 and 0.4.1 is pending.

**Pull request merged**